src/CRUD.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-restricted-globals */
+import Icon from './icons8-drop-down-24.png';
+import removeIcon from './icons8-remove-24.png';
+import editIcon from './icons8-edit-24.png';
+import './style.css';
+
+export default class Task {
+  constructor(index, description, completed) {
+    this.index = index;
+    this.description = description;
+    this.completed = completed;
+  }
+
+  taskList = [];
+
+  editMood = false;
+
+  add() {
+    const temp = new Task(this.taskList.length, document.getElementById('inputDescription').value, false);
+    this.taskList.push(temp);
+    localStorage.setItem('taskList', JSON.stringify(this.taskList));
+    this.taskList = JSON.parse(localStorage.getItem('taskList'));
+    this.update();
+    location.reload();
+  }
```

Comment:
Please ensure that `items` are numbered from ` 1 ` according to the project requirement and NOT ` 0 `.
![image](https://user-images.githubusercontent.com/10332499/155408148-57373098-b3c5-4ed1-8e2b-b2600dc36fab.png).